lib/backend/registrybackend/security/security.go

```diff
@@ -97,9 +97,12 @@ func (a *authenticator) Authenticate(repo string) ([]httputil.SendOption, error)
 		return opts, nil
 	}
 
-	if !config.EnableHTTPFallback {
+	if config.EnableHTTPFallback {
```

Comment:
I guess it is because we always create the option with httpFallbackDisabled as true.
https://sourcegraph.com/github.com/uber/kraken@master/-/blob/utils/httputil/httputil.go?L290

Reply:
Yes we are always doing Fallback Disabled and never enabling it.